src/styles/_catalog.scss

```diff
@@ -1,9 +1,23 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, auto);
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+    grid-template-columns: repeat(3, minmax(200px, auto));
+  }
+
+  @media (min-width: 488px) and (max-width: 767px) {
+    grid-template-columns: repeat(2, minmax(200px, auto));
+  }
+
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, minmax(200px, auto));
+  }
```

Comment:
U can use only 1 media query, try to use autofill

Reply:
fixed))